src/test/scala/dev/kamu/engine/spark/ingest/EngineIngestTest.scala

```diff
@@ -252,4 +260,71 @@ class EngineIngestTest extends FunSuite with KamuDataFrameSuite with Matchers {
     )
   }
 
+  test("ingest Parquet") {
+    Temp.withRandomTempDir("kamu-engine-spark")(
+      tempDir => {
+        val outputLayout = tempLayout(tempDir, "out")
+
+        val inputPath = tempDir.resolve("input1.parquet")
```

Comment:
We also wanted to check what happens when the input file has no extension and is compressed/uncompressed - whether Spark will be able to deduce the type and do the right thing.

I suspect it won't ... so we'll need a workaround, since `kamu-cli` will map input file into container as `read.bin`.

Reply:
Both file without any extension or `read.bin` are accepted without issues.

Reply:
I've tried experimenting with `CompressionCodec`. Our utility to write Parquet supports `GZIP`, `Snappy`, and `Uncompressed`. Other methods in the enum require downloading some extras libraries. Anyway, Spark normally ingests from files with whatever codec was used to compress those. I've left `read.bin` and uncompressed version in the latest edition of the test.